ruleset/rules/0920-oracledb_rules.xml

```diff
@@ -1,31 +1,24 @@
-    -->
-    <rule id="89100" level="3">
-        <decoded_as>oracledb_log</decoded_as>
-        <description>OracleDB transaction</description>
-        <group>pci_dss_10.6.2,hipaa_164.312.b</group>
-    </rule>
+  <rule id="89100" level="3">
+    <decoded_as>oracledb-log</decoded_as>
+    <description>OracleDB transaction.</description>
+    <group>hipaa_164.312.b,pci_dss_10.6.2,</group>
+  </rule>
 
-    <!--
-      Alert example:
-      {"timestamp":"2021-07-14T11:20:43.107-0700","rule":{"level":7,"description":"Oracle DB alerts","id":"89101","firedtimes":1,"mail":false,"groups":["oracle","oracledb"],"pci_dss":["10.6.1"],"hipaa":["164.312.b"]},"agent":{"id":"000","name":"localhost.localdomain"},"manager":{"name":"localhost.localdomain"},"id":"1626286843.176665","full_log":"Jul 14 11:20:43 localhost.localdomain oracledb_alerts: Tue Feb 23 11:53:46 2021\anyword: mtype: 61 process 70621 failed because of a resource problem in the OS. The OS has most likely run out of buffers (rval: 4)\nErrors in file /opt/oracle/app/oracle/diag/rdbms/cdbtst00/CDBTST000/trace/CDBTST000_m000_10111.trc  (incident=346445):\nORA-00603: ORACLE server session terminated by fatal error\nORA-27504: IPC error creating OSD context\nORA-27300: OS system dependent operation:sendmsg failed with status: 105\nORA-27301: OS failure message: No buffer space available\nORA-27302: failure occurred at: anyword\nopidrv aborting process M000 ospid (70621) as a result of ORA-603\nProcess m000 died, see its trace file","predecoder":{"program_name":"oracledb_alerts","timestamp":"Jul 14 11:20:43","hostname":"localhost.localdomain"},"decoder":{"name":"oracledb_alerts"},"location":"/tmp/oracle/alert_CDBTST931.log"}    
-    -->
-    <rule id="89101" level="7">
-      <decoded_as>oracledb_alerts</decoded_as>
-      <description>Oracle DB alerts</description>
-      <group>pci_dss_10.6.1,hipaa_164.312.b</group>
+  <rule id="89101" level="7">
```

Comment:
Why is this a level 7 alert?

Reply:
They include words like “bad”, “error”, etc.
These events are most of the time unclassified and may have some security relevance.

There is no clue about alert type, so we use level 7 to identify any possible security relevance from oracle alert.